### PR TITLE
Implement anyhow

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/commands/keywords.rs [hunk 1]
```diff
@@ -114,11 +110,11 @@ pub async fn add(
 		Some(captures) => {
 			let keyword = captures
 				.get(1)
-				.ok_or("Captures didn't contain keyword")?
+				.ok_or_else(|| anyhow!("Captures didn't contain keyword"))?
```
Comment (ghost) [inline src/commands/keywords.rs]: [The function will always be called and potentially allocate an object acting as the default.](https://rust-lang.github.io/rust-clippy/master/index.html#or_fun_call)

Comment (ThatsNoMoon) [reply src/commands/keywords.rs]: Can these use `.context("message")` instead? [`Option::context`](https://docs.rs/anyhow/1.0.37/src/anyhow/context.rs.html#83-88) is essentially equivalent to this.

Comment (ghost) [reply src/commands/keywords.rs]: Sure

File: src/commands/keywords.rs [hunk 2]
```diff
@@ -3,12 +3,12 @@
 
 //! Commands for adding, removing, and listing keywords.
 
-use anyhow::{anyhow, Result};
+use anyhow::{Context, Result};
```
Comment (ThatsNoMoon) [inline src/commands/keywords.rs:6]: This can be
```rs
use anyhow::{Context as _, Result};
```
to avoid conflicting with `serenity::client::Context`.

**Review by ThatsNoMoon — CHANGES_REQUESTED** [review]

**Pull request merged**